recipes/atlas-gene-annotation-manipulation/build.sh

```diff
@@ -0,0 +1,2 @@
+mkdir -p $PREFIX/bin
+cp *.sh *.R $PREFIX/bin
```

Comment:
are all these guys chmoded to be executable?

Reply:
Yep!